api/src/pcapi/core/finance/models.py

```diff
@@ -186,6 +186,13 @@ class Cashflow(Model):
     creationDate = sqla.Column(sqla.DateTime, nullable=False, server_default=sqla.func.now())
     status = sqla.Column(db_utils.MagicEnum(CashflowStatus), index=True, nullable=False)
 
+    # FIXME (dbaty, 2022-01-26): set NOT NULL constraint once the
```

Comment:
Est-ce qu'une contrainte `NOT NULL` peut être créée dès maintenant`NOT VALID` — donc ne concerner que les `INSERT` ou `UPDATE` et pas les rangs existants —, en prévoyant d'ajouter une migration de validation une fois le rattrapage effectué. Cela permettrait de repérer dès maintenant les endroits qui ne définissent pas `cashflow.busineesUnitId`.

Reply:
C'est une bonne idée, mais je dois avouer que je ne suis pas très motivé pour faire ça la tout de suite... J'y penserai pour la prochaine fois.